lib/ios-deploy.js

```diff
@@ -23,6 +23,7 @@ const APP_INSTALL_STRATEGY = Object.freeze({
 class IOSDeploy {
```

Comment:
It would make sense to rename this class if possible. For example to `RealDevice`

Reply:
Yes, agree. I'd like to do it as a separate PR to not combine it into this, though.

Reply:
-> https://github.com/appium/appium-xcuitest-driver/pull/2353

Reply:
Please merge https://github.com/appium/appium-xcuitest-driver/pull/2353 first and rebase this PR with it

Reply:
done